opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -38,6 +38,7 @@
 )
 
 LICENSE_URL = "https://github.com/opensearch-project/opensearch-py-ml/raw/main/LICENSE"
+THIRD_PARTY_FILE_NAME = "THIRD-PARTY"
```

Comment:
Could you please check with the open source engineer that if naming the attribution file as `THIRD-PARTY` looks good to him or not?

Reply:
The name `THIRD_PARTY` is provided by the open source engineer